.github/workflows/aws-cleanup.yaml

```diff
@@ -0,0 +1,25 @@
+name: AWS cleanup
+
+on:
+  schedule:
+    - cron: '0 0 * * 0'
+  workflow_dispatch: {}
+
+jobs:
+  cleanup:
+    name: Cleanup AWS resources
+    environment: "cleanup AWS"
+    runs-on: ubuntu-latest
+    container: rebuy/aws-nuke:v2.15.0-rc.3
```

Comment:
EKSNodegroups and EKSFargate only got added in `2.15.0` which only has RC's out. Also annoyingly no `stable` tag that can be used

Reply:
what's the worst bug the RC can have? it deletes some resources?

Reply:
yeah. Although all of my manual testing with this has worked, so I think the risks are low.